cmd/tlsproxy/main.go

```diff
@@ -0,0 +1,98 @@
+package main
```

Comment:
I took the liberty of moving this into `cmd/tlsproxy` so that `go install` installs something called `tlsproxy` instead of `cmd`.